src/components/Restoration/BusinessType.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
New component that we didn't notice in the uxpin. Fortunately it's a simple one.